src/main/java/org/opensearch/security/support/SafeSerializationUtils.java

```diff
@@ -48,6 +49,7 @@ public final class SafeSerializationUtils {
         InetSocketAddress.class,
         Pattern.class,
         User.class,
+        PluginUser.class,
```

Comment:
Tested this change out by making the SystemIndexTests use a 3 node cluster (1 manager + 2 data nodes) and then verifying that the [deserialized User object](https://github.com/opensearch-project/security/blob/main/src/main/java/org/opensearch/security/transport/SecurityRequestHandler.java#L184-L187) returned `true` when `User.isPluginUser()` was called.